src/app/server/CommissioningWindowManager.cpp

```diff
@@ -47,6 +47,10 @@ void OnPlatformEventWrapper(const chip::DeviceLayer::ChipDeviceEvent * event, in
 
 namespace chip {
 
+#if defined(CONFIG_IM_BUILD_FOR_UNIT_TEST) || defined(CHIP_CONFIG_TESTS)
+Optional<System::Clock::Seconds16> CommissioningWindowManager::mMinCommissioningTimeoutOverride;
```

Comment:
This is not a `mXXXX` (member), it's `gXXXX` global. This should be behind a setter function